python/cudf/cudf/_libxx/lib.pxd

```diff
@@ -14,18 +14,14 @@ from libcpp.pair cimport pair
 
 from rmm._lib.device_buffer cimport device_buffer, DeviceBuffer, move
 
+ctypedef bool underlying_type_t_order
+ctypedef bool underlying_type_t_null_order
+ctypedef bool underlying_type_t_sorted
```

Comment:
If we're using this pattern could you do the same for the `mask_state` underlying_type as well?

Reply:
Yes. Question, though. Right now the Enums are accessed as `libcudfxx.lib.MyEnum.FOO`. Should we expose those as `libcudfxx.MyEnum.FOO`?

Reply:
I don't think any of them will be used publicly, but they will be used somewhat commonly by developers within cuDF. Given `libcudfxx` is internal only as well I'm fine with exposing them for developer convenience.

Reply:
I kept `mask_state` in `null_mask.pyx`, as `_lib` was depending on it (not sure why), and moving it created an import issue that wasn't easily identifiable, potentially due to a circular reference.

Reply:
leaving this as is since it won't be used publicly.